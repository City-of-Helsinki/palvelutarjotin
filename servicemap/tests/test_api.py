GET_SCHOOLS_AND_KINDERGARTENS_LIST_QUERY = """
  query ServicemapSchoolsAndKindergartensList {
    schoolsAndKindergartensList {
      meta {
        count
        next
        previous
      }
      data {
        id
        name {
          fi
          sv
          en
        }
      }
    }
  }
  """


def test_list_helsinki_schools_and_kindergartens(
    api_client, snapshot, mock_get_servicemap_schools_and_kindergartens_data
):
    executed = api_client.execute(GET_SCHOOLS_AND_KINDERGARTENS_LIST_QUERY)
    snapshot.assert_match(executed)
