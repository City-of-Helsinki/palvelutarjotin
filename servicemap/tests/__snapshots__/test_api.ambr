# serializer version: 1
# name: test_list_helsinki_schools_and_kindergartens
  dict({
    'data': dict({
      'schoolsAndKindergartensList': dict({
        'data': list([
          dict({
            'id': 'tprek:3',
            'name': dict({
              'en': 'Daycare Kannel',
              'fi': 'Päiväkoti Kannel',
              'sv': 'Päiväkoti Kannel',
            }),
          }),
          dict({
            'id': 'tprek:2266',
            'name': dict({
              'en': 'Family day care Itä-Kannelmäki - Hakuninmaa - Maununneva',
              'fi': 'Perhepäivähoito Kaarela',
              'sv': 'Perhepäivähoito Kaarela',
            }),
          }),
          dict({
            'id': 'tprek:73213',
            'name': dict({
              'en': 'Daycare Kannelmäki, preschool groups',
              'fi': 'Päiväkoti Kannelmäki, esiopetusryhmät',
              'sv': 'Päiväkoti Kannelmäki, förskolegrupper',
            }),
          }),
        ]),
        'meta': dict({
          'count': 3,
        }),
      }),
    }),
  })
# ---
